Add update_check config reference (#6673)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/global.rst

| **Allowed values** | A positive number between 1 and 3600|
+--------------------+-------------------------------------+

.. _reference_update_check:

update_check
^^^^^^^^^^^^

> This toggles whether to query the external Wazuh CTI service to retrieve the available Wazuh updates.

Reviewer: ```suggestion
This setting toggles whether to query the external Wazuh Cyber Threat Intelligence (CTI) service for any available Wazuh updates.
```

> This toggles whether to query an external service to retrieve any available update from the dashboard.

Reviewer 2: ```suggestion
This toggles whether to query the external Wazuh CTI service to retrieve the available Wazuh updates.
```
